build.gradle

```
@@ -11,10 +11,10 @@ buildscript {
   }
 
   dependencies {
-    classpath deps.plugins.kotlin
-    classpath deps.plugins.android
-    classpath deps.plugins.versions
-    classpath deps.plugins.spotless
+    classpath(libs.kotlinGradlePlugin)
+    classpath(libs.androidGradlePlugin)
+    classpath(libs.versionsGradlePlugin)
+    classpath(libs.spotlessGradlePlugin)
```

```suggestion
    classpath libs.kotlinGradlePlugin
    classpath libs.androidGradlePlugin
    classpath libs.versionsGradlePlugin
    classpath libs.spotlessGradlePlugin
```

same with all the other configurations in the other build.gradles

---

for plugins, let's go with `classpath libs.plugins.spotless`

---

I can come up with why for this, but I'm curious what's your answer, why remove the parentheses?

---

fixed in https://github.com/cashapp/paparazzi/pull/571/commits/d84ebe01d676b24721fb7898d913bcb3644b6964 on #571, but still curious why